src/component/calculator.js

```diff
@@ -0,0 +1,41 @@
+/* eslint-disable react/prefer-stateless-function */
+import React from 'react';
+
+class Calculator extends React.Component {
+  render() {
+    return (
+      <div className="calculator">
+        <div className="display-section">
+          <p className="diplay-value">0</p>
+        </div>
+
+        <button type="button">AC</button>
+        <button type="button">+/-</button>
+        <button type="button">%</button>
+        <button type="button" className="operator">÷</button>
+
+        <button type="button">7</button>
+        <button type="button">8</button>
+        <button type="button">9</button>
+        <button type="button" className="operator">x</button>
+
+        <button type="button">4</button>
+        <button type="button">5</button>
+        <button type="button">6</button>
+        <button type="button" className="operator">-</button>
+
+        <button type="button">1</button>
+        <button type="button">2</button>
+        <button type="button">3</button>
+        <button type="button" className="operator">+</button>
+
+        <button type="button" className="zero">0</button>
+        <button type="button">.</button>
+        <button type="button" className="operator">=</button>
+
+      </div>
```

Comment:
- [ ] You did a great job developing `Calculator.js` :+1:, but as per the project requirements, you need to create at least one child component for `Calculator.js`.

Reply:
Thanks for your feedback! I understand that I need to create a child component for Calculator.js as per project requirements, and I'll get on it right away. 😊